sdk/storage/Azure.Storage.Common/src/Shared/Constants.cs

```diff
@@ -70,7 +70,10 @@ internal static class Constants
 
         // SASTimeFormat represents the format of a SAS start or expiry time. Use it when formatting/parsing a time.Time.
         // ISO 8601 uses "yyyy'-'MM'-'dd'T'HH':'mm':'ss"
-        public const string SasTimeFormat = "yyyy-MM-ddTHH:mm:ssZ";
+        public const string SasTimeFormatSeconds = "yyyy-MM-ddTHH:mm:ssZ";
+        public const string SasTimeFormatSubSeconds = "yyyy-MM-ddTHH:mm:ss.fffffffZ";
```

Comment:
Should we support other fractional counts? if yes then we should change fffffffZ to FFFFFFFZ.

Reply:
I was experimenting with this, and it looks like the service expects 7 digits of specificity.